test_scripts/RC/InteriorVehicleData_cache/000_cache_2_apps_sequence.lua

```diff
@@ -6,6 +6,27 @@
 -- Requirement summary: TBD
 --
 -- Description:
+-- 1. App1 sends GetInteriorVD(module_1) without subscribe parameter
+-- 2. App2 sends GetInteriorVD(module_1) without subscribe parameter
+-- 3. App1 sends GetInteriorVD(module_1) with subscribe=true
+-- 4. HMI sends OnInteriorVD(data for module_1)
```

Comment:
@GetmanetsIrina After step 4 it is missing step: `App1 -> GetInteriorVehicleData with subscribe=true` which is in Scenario